requirements.yml

```diff
@@ -3,3 +3,4 @@ roles:
   - geerlingguy.postgresql
 collections:
   - ansible.posix
+  - mdellweg.filters
```

Comment:
I see you have special logic for the jq filter, and that we do not use the jq filter in pulp_installer.

Just to confirm, there are no other python dependencies this would introduce, correct?

Reply:
Also, I feel we should add a changelog entry for this overall change. A new dependency is worth mentioning, even if we do not support running from the pulp_installer dir anymore., and even if we require Ansible 2.9 already anyway.